TrainingTests/TestUtils/Stubs.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Stubs.swift
+//  TrainingTests
+//
+//  Created by Ong Yue Huei on 19/07/2021.
+//
+
+import Foundation
+@testable import Training
+
+extension String {
+    static func stub() -> Self {
+        UUID().uuidString
+    }
+}
+
+extension Int {
+    static func stub() -> Self {
+        Int.random(in:100..<999)
+    }
+}
+
+extension Int64 {
+    static func stub() -> Self {
+        Int64(Int.stub())
+    }
+}
+
+extension URL {
+    static func stub() -> Self {
+        URL(string: .stub())!
+    }
```

Comment:
`URL`はランダムなstringだとnilになる場合があるので、以下のような形にすると良いと思います！
```suggestion
    static func stub() -> Self {
        URL(string: "https://www.google.com/search?q=\(UUID().uuidString)")!
    }
```

Reply:
承知しました！教えていただきありがとうございます！ 🙇‍♀️🙇‍♀️